apps/web/e2e/unit-tests-converted/__tests__/components/export-dialog.test.tsx

```
@@ -0,0 +1,504 @@
+  Label: ({ children, htmlFor }: any) => React.createElement('label', { htmlFor, 'data-testid': 'label' }, children)
+}))
+
+jest.mock('@/components/ui/button', () => ({
+  Button: ({ children, onClick, disabled, variant, className }: any) => 
+    React.createElement('button', {
+      onClick,
+      disabled,
+      'data-testid': 'button',
+      'data-variant': variant,
+      className
+    }, children)
```

Biome (2.1.2)

[error] Provide a valid type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)